afs-ext-base/src/main/java/com/powsybl/afs/ext/base/ImportedCase.java

```diff
@@ -79,6 +80,11 @@ public Network getNetwork() {
         return findService(NetworkCacheService.class).getNetwork(this);
     }
 
+    @Override
+    public Network getNetwork(NetworkListener listener) {
```

Comment:
We must add some unit test : maybe just check that listeners have indeed been registered with the returned network ?

Reply:
ok